tests/versioned/package.json

```diff
@@ -5,18 +5,18 @@
   "tests": [
     {
       "engines": {
-        "node": ">=4"
+        "node": "4 || >5"
```

Comment:
Wondering if should indicate (via the notes here as well?) why Node 5 doesn't work so don't have to remember the DEBUG dependency thing.